redbot/__main__.py

```diff
@@ -10,6 +10,22 @@
 
 import discord
 
+# Set the event loop policies here so any subsequent `get_event_loop()`
+# calls, in particular those as a result of the following imports,
+# return the correct loop object.
+if sys.platform == "win32":
+    asyncio.set_event_loop_policy(asyncio.WindowsProactorEventLoopPolicy())
+else:
+    # Let's not force this dependency, uvloop is much faster on cpython
+    if sys.implementation.name == "cpython":
```

Comment:
Mostly nitpicking, but I'd prefer this converted to an `elif` for the indentation to properly represent the options as "on windows, not on windows, but on cpython, something else.